### PR TITLE
Add TFSlim

### PR DESCRIPTION
**Pull request by cthacker**
• adds tfslim runner/trainer which allows for using any of the pre-trained models
• allows for tuning of whole model during transfer learning by setting tune_full_net=True in train()
• cleaned up some code
• still TODO: validation/test set accuracy in training benchmark

### CODE REVIEW

**Comment by dominiek** [comment]
Wow dude badass! And here I thought you were just going to commit some whitespace : - ]

Thanks!
